.github/workflows/build-ci-buildbox-images.yaml

```diff
@@ -7,14 +7,14 @@ on:
       - build.assets/Dockerfile
       - build.assets/Dockerfile-arm
       - build.assets/Dockerfile-centos7
-      - build.assets/Dockerfile-centos7-fips
       - build.assets/Dockerfile-node
       - build.assets/Makefile
       - build.assets/images.mk
       - build.assets/versions.mk
     branches:
       - master
       - branch/**
+      - camh/*
```

Comment:
Friendly reminder not to forget to remove this prior to merge.

Reply:
First thing I thought of when I woke up :)